metrics/ci_integrations/lib/source/buildkite/config/validator/buildkite_source_validator.dart

```diff
@@ -46,25 +44,25 @@ class BuildkiteSourceValidator
     final auth = BearerAuthorization(accessToken);
 
     final authValidationResult = await validationDelegate.validateAuth(auth);
-    validationResultBuilder.setResult(
-      BuildkiteSourceConfigField.accessToken,
-      authValidationResult,
-    );
+    validationResultBuilder.setResult(authValidationResult);
 
-    if (authValidationResult.isFailure) {
+    if (authValidationResult.conclusion ==
+        ConfigFieldValidationConclusion.invalid) {
```

Comment:
🔨 
What do you think about adding the `isFailure` getter as it was in the `FieldValidationResult` class?